Week1/homework/js-exercises/ex7.js

```diff
@@ -0,0 +1,7 @@
+//Exercise 7: Log the length of a string
+//eclare a variable
+let mySentence = 'Programming is so interesting!';
```

Comment:
Ah, is it better to use `const` here, or `let` :)?

Reply:
yes it's better, changes done locally.